src/application/i18n/messages/en.json

```diff
@@ -10,10 +10,14 @@
       "noteSettings": "Settings"
     }
   },
+  "confirmations": {
+    "uninstallEditorTool": "Do you really want to delete this tool?"
+  },
```

Comment:
Let's move it to the `userSettings.toolUninstallConfirmation`

Reply:
done